unstructured/partition/pdf.py

```diff
@@ -49,9 +49,10 @@ def partition_pdf(
     infer_table_structure: bool = False,
     ocr_languages: str = "eng",
     max_partition: Optional[int] = 1500,
+    min_partition: Optional[int] = 0,
     include_metadata: bool = True,
     metadata_filename: Optional[str] = None,
-    min_partition: Optional[int] = 0,
```

Comment:
Can we keep the kwarg order the same here for backward compatibility?

Reply:
Do you mean to leave `min_partition` in  54 line?

Reply:
> Can we keep the kwarg order the same here for backward compatibility?

Do you mean to leave min_partition in 54 line?